imagedephi/redact/redact.py

```diff
@@ -55,11 +47,22 @@ def iter_image_files(directory: Path) -> Generator[Path, None, None]:
             yield child
 
 
+def create_redact_dir(base_output_dir: Path) -> Path:
+    time_stamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    redact_dir = base_output_dir / f"Redacted_{time_stamp}"
+    try:
+        redact_dir.mkdir(parents=True)
+    except PermissionError:
+        "Cannnot create an output directory, permission error."
```

Comment:
What's this line doing? It's just a string.

Reply:
I think i accidentally removed the click.echo :woman_facepalming:  fixed in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf